traja/models/train.py

```diff
@@ -245,15 +246,27 @@ def fit(
                     )
                 )
 
-            # Testing
-            if epoch % 10 == 9 and epoch != 0:
+            # Testing & Validation
+            evaluate_for_this_epoch = False
+            data_loader_to_evaluate = validation_loader
+            current_set = "Validation"
+            if epoch % validate_every == validate_every - 1 and epoch != 0:
+                data_loader_to_evaluate = validation_loader
+                evaluate_for_this_epoch = True
+                current_set = "Validation"
+            if epoch % test_every == test_every - 1 and epoch != 0:
```

Comment:
You don't have to submit a validation set. With the current defaults one would trigger a crash. There should be a check to make sure there is a validation set present.

Reply:
Done